scrapy/extensions/feedexport.py

```diff
@@ -140,6 +140,8 @@ class FeedExporter(object):
 
     def __init__(self, settings):
         self.settings = settings
+        self.indexfmt = '{:05d}'
```

Comment:
I think we shouldn't hardcode format for the index; users can write `/tmp/%(start)s/feed-%(index)05d.json` to get the same result.